index.md

```diff
@@ -138,11 +138,11 @@ So, we can see that we have several files, directories and a program. Great!
 
 Most programs take additional arguments that control their exact
 behavior. For example, `-F` and `-l` are arguments to `ls`.  The `ls`
-program, like many programs, take a lot of arguments. But how do we
+program, like many programs, takes a lot of arguments. But how do we
 know what the options are to particular commands?
 
 Most commonly used shell programs have a manual. You can access the
-manual using the `man` program. Try entering:
+manual using the `man` program. The man program will display the manual pages of the ls command page by page; use the spacebar to get tot the next page. Try entering:
```

Comment:
to keep the formatting as  in the rest of the document is preferred to break your line into three lines.